### PR TITLE
PM-1895 - Add Matterbridge-bot helm chart

### PR DESCRIPTION
**Pull request by kaozenn**

### CODE REVIEW

File: matterbridge-bot/README.md
Comment (kaozenn) [inline matterbridge-bot/README.md:1]: I need to run pre-commit to populate it. I have having issues with it at the moment

Comment (johnmarcou) [reply matterbridge-bot/README.md:1]: If the hook did not run on the changed files when committing, make sure your hooks are locally installed: `pre-commit install`.

You can force run the hooks on all files with `pre-commit run --all-files`.

**Review by johnmarcou — APPROVED** [review]
LGTM

**Review by simisimis — APPROVED** [review]
Looks good! :clap:

**Pull request merged**